web/src/lightbox.js

```diff
@@ -494,15 +550,24 @@ export function initialize() {
 
     $("#lightbox_overlay").on("click", ".image-list .image", function () {
         const $image_list = $(this).parent();
-        const $original_image = $(
-            `.message_row img[src='${CSS.escape($(this).attr("data-src"))}']`,
-        );
+        let $original_image;
+        const is_video = $(this).hasClass("video");
+        if (is_video) {
+            $original_image = $(
+                `.message_row video[src='${CSS.escape($(this).attr("data-src"))}']`,
+            );
+        } else {
+            $original_image = $(`.message_row img[src='${CSS.escape($(this).attr("data-src"))}']`);
+        }
 
         open_image($original_image);
 
+        if (!$(".image-list .image.selected").hasClass("video") || !is_video) {
+            pan_zoom_control.reset();
+        }
```

Comment:
Does this logic need to explicitly hide those controls for video elements (i.e. the else case here), or is that done elsewhere?

Reply:
It is actually just trying to reset the zoom level to normal for videos since we don't support pan zoom for videos.

The zoom button is displayed / hidden at the top of `display_image / video`.